content/en/examples/application/mysql/mysql-statefulset.yaml

```diff
@@ -17,7 +17,7 @@ spec:
     spec:
       initContainers:
       - name: init-mysql
-        image: mysql:5.7
+        image: mysql:5.7-debian
```

Comment:
Why is this the right change to make?

Reply:
See https://github.com/docker-library/mysql/issues/877 (sorry I wanted to add this in the commit/description as well).